core/src/main/java/com/testaem/aem/core/utills/Truncator.java

```diff
@@ -0,0 +1,5 @@
+package com.testaem.aem.core.utills;
+
+public interface Truncator {
```

Comment:
Important!
We do not create interfaces as a religion.
Interfaces are only needed when they are needed. 
Interfaces are only needed when there is a contract that can have several implementations, and we do not know (and don't give a damn) what implementation is we're working with.

Interfaces are good and justified for Services. Because services are such, that you can override an OOTB implementation with an implementation of your own.

Interfaces are *only barely* good for Sling models. A Sling model is usually just a plain object, POJO. Usually it does not have any variants. But... well... it might have some. So interface is not without all of sense.

But interface for a utility class is an overkill)